controller/workitem_blackbox_test.go

```diff
@@ -504,6 +506,23 @@ func createOneRandomArea(ctx context.Context, db *gorm.DB) *area.Area {
 	return &ar
 }
 
+func newChildIteration(ctx context.Context, db *gorm.DB, parentIteration *iteration.Iteration) *iteration.Iteration {
+	iterationRepo := iteration.NewIterationRepository(db)
+
+	parentPath := append(parentIteration.Path, parentIteration.ID)
+	itr := iteration.Iteration{
+		Name:    "Sprint 101",
+		SpaceID: parentIteration.SpaceID,
+		Path:    parentPath,
+	}
+	err := iterationRepo.Create(ctx, &itr)
+	if err != nil {
+		fmt.Println("Failed to create iteration.")
```

Comment:
@sbose78 thanks, I did not want these helper methods to actually assert or test. Hence the caller is checking if instance is created or not like this - https://github.com/almighty/almighty-core/pull/1008/files#diff-98914f859911696f3cdd98dd289d00faR2038
Will this do ?